core/src/js/components/mercenaries/overlay/teams/mercenaries-team-ability.component.ts

```diff
@@ -70,6 +70,7 @@ import { CardsFacadeService } from '../../../../services/cards-facade.service';
 export class MercenariesTeamAbilityComponent {
 	@Input() tooltipPosition: boolean;
 	@Input() uncapacitated: boolean;
+	@Input() side: 'player' | 'opponent' | 'out-of-combat-player';
```

Comment:
Here I would just pass a `enableHighlight` boolean, so that the ability component doesn't have the logic itself (it's the kind of logic I'd like to keep as high as possible in the components chain)